### PR TITLE
[ALS-5579] Fix the release branch of the all-in-one

### PR DESCRIPTION
**Pull request by Luke-Sikina**

### CODE REVIEW

File: initial-configuration/install-dependencies-docker.sh [hunk 1]
```diff
@@ -3,30 +3,34 @@
...
 CWD=`pwd`
 
-# $1 is the path to the docker-config directory $2 is the path to the rc file
-function set_docker_config_dir {
-  local dir=$1
-  local file=$2
-  if [ -z "$dir" ]; then
-   dir="/var/local/docker-config"
+# $1 is the path to the docker-config docker_config_directory $2 is the path to the rc rc_file
+function set_docker_config_docker_config_dir {
```
Comment (JamesPeck) [inline initial-configuration/install-dependencies-docker.sh:7]: Why not set_docker_config_dir?

Comment (Luke-Sikina) [inline initial-configuration/install-dependencies-docker.sh:7]: fix renamed

File: initial-configuration/install-dependencies-docker.sh [hunk 2]
```diff
@@ -79,11 +83,11 @@ if [[ "$OSTYPE" =~ ^darwin ]]; then
     echo $1
     #check for $1 arg
     if [ -z "$1" ]; then
-      echo "No arguments supplied. Please provide the path to the docker-config directory."
-      echo "MacOS doesn't like the default docker-config dir Please supply a directory as an arguments."
+      echo "No arguments supplied. Please provide the path to the docker-config docker_config_directory."
+      echo "MacOS doesn't like the default docker-config docker_config_dir Please supply a docker_config_directory as an arguments."
```
Comment (Luke-Sikina) [inline initial-configuration/install-dependencies-docker.sh:87]: dur

File: initial-configuration/mysql-docker/docker-compose.yml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 version: '3'
 services:
   picsure-db:
-    image: mysql:8.0
+    image: mysql:5.7
```
Comment (Luke-Sikina) [inline initial-configuration/mysql-docker/docker-compose.yml:5]: 8.0

**Review by srpiatt — APPROVED** [review]
I am approving this with faith that it works. XD

**Pull request merged**